pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1391,4 +1391,9 @@ self: super: {
   gi-soup = doJailbreak super.gi-soup;
   gi-webkit2 = doJailbreak super.gi-webkit2;
 
+  # Upstream PR: https://github.com/jkff/splot/pull/9
+  splot = appendPatch (unmarkBroken super.splot) (pkgs.fetchpatch {
```

Comment:
@basvandijk Thanks for trying to fix this up.

However, instead of using `unmarkBroken` here, can you rebase this change on the `haskell-updates` branch, and remove `splot` from the list of `broken-packages` in `configuration-hackage2nix.yaml`?

Reply:
Sure thing.

Reply:
@cdepillabout done.